packages/rrweb/src/replay/index.ts

```diff
@@ -572,9 +616,11 @@ export class Replayer {
     this.wrapper.classList.add('replayer-wrapper');
     this.config.root.appendChild(this.wrapper);
 
-    this.mouse = document.createElement('div');
-    this.mouse.classList.add('replayer-mouse');
-    this.wrapper.appendChild(this.mouse);
+    for (let [, { pointerEl }] of Object.entries(this.pointers)) {
+      pointerEl = document.createElement('div');
+      pointerEl.classList.add('replayer-mouse');
+      this.wrapper.appendChild(pointerEl);
+    }
```

Comment:
I don't think this is doing much since `this.pointers` will be empty when this is called? I think what you could do is always create the DOM node for a pointerId = -1 to maintain the previous behavior.

Reply:
ah you're right, i'll replace with `createPointer(-1)`